DashKitUI/View Controllers/DashUICoordinator.swift

```diff
@@ -182,7 +183,13 @@ class DashUICoordinator: UINavigationController, PumpManagerSetupViewController,
             if let pumpManager = pumpManager {
                 let vc = PodSetupCompleteViewController.instantiateFromStoryboard(pumpManager, navigator: self)
                 vc.completion = { [weak self] in
-                    self?.stepFinished()
+                    if let self = self {
+                        let settings = PumpManagerSetupSettings(maxBasalRateUnitsPerHour: self.maxBasalRateUnitsPerHour,
+                                                           maxBolusUnits: self.maxBolusUnits,
+                                                           basalSchedule: self.basalSchedule)
+                        self.pumpManagerOnboardDelegate?.pumpManagerOnboardNotifying(didOnboardPumpManager: pumpManager, withFinalSettings: settings)
```

Comment:
I'd feel more comfortable it @ps2 ok'd this as well...

Reply:
Hmm, I missed this in the LoopKit protocol changes. PumpManager setup should not be the component responsible for defining max basal/bolus or basal schedule.  That should be set during therapy settings setup DIY onboarding does it in that order, and I think Tidepool is already set up to do it in that order as well, so I *think* this can be cleaned up now.
The chicken and egg problem is solved like this; 1) the user *selects* the devices at the start of onboarding, but does not go through onboarding for those devices yet.  2) the selection of the pump is enough for the main onboarding to call static methods on the pump manager to get pump delivery resolution. 3) After the user has configured basal rates, max basal/bolus, those settings are passed to the pump manager onboarding.

Reply:
This code is simply mirroring what was there, but does it via the `PumpManagerSetupSettings` struct instead of a protocol with `{ get set }` properties. I do not think it is the right time to change how this works. 

Created Jira ticket to capture this work for later:  https://tidepool.atlassian.net/browse/LOOP-3112